ui/src/app/nav.tsx

```diff
@@ -107,19 +24,25 @@ export function SideBar({ sidebarVisible, setSidebarVisible }: { sidebarVisible:
                             <nav aria-label="Main">
                                 <ul className="p-side-navigation__list">
                                     <li className="p-side-navigation__item">
-                                        <a className="p-side-navigation__link" href="/certificate_requests" aria-current={activeTab === "certificate_requests" ? "page" : "false"} >
+                                        <a className="p-side-navigation__link" href="/certificate_requests" aria-current={activePath.startsWith("/certificate_requests") ? "page" : "false"} >
                                             <i className="p-icon--security is-light p-side-navigation__icon"></i>
                                             <span className="p-side-navigation__label">
                                                 <span className="p-side-navigation__label">Certificate Requests</span>
```

Comment:
The Certificate Requests menu option on the left side navigation is not a clickable button, in the sense that the mouse does not switch to a finger when we hover on top of it? Was this the case prior to this PR?

Reply:
I just checked main, and it looks like the mouse only changes when the tab isn't selected. I've added manual styling so that it always appears on both tabs.